mmv1/third_party/terraform/tests/resource_compute_instance_template_test.go.erb

```diff
@@ -3127,8 +3135,9 @@ resource "google_compute_instance_template" "foobar" {
 `, suffix)
 }
 
-func testAccComputeInstanceTemplate_sourceSnapshotEncryptionKey(kmsRingName, kmsKeyName, suffix string) string {
-	return fmt.Sprintf(`
+func testAccComputeInstanceTemplate_sourceSnapshotEncryptionKey(context map[string]interface{}) string {
+	# kmsRingName, kmsKeyName, suffix string
+	return Nprintf(`
```

Comment:
When switching to `Nprintf`, I think you'll need to change all instances of `%s` in these blocks to their named counterpart, ie. `%{kms_ring_name}`

Reply:
Should be good now